medical_physician/models/medical_physician_service.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class MedicalPhysicianService(models.Model):
+    """ Services provided by the Physician at a specific medical center.
+
+    A physician could have "surgeries" on one center but only
```

Comment:
I think we we should bring this kind of information in the README where everyone (including non-code-reader) can take advantage of it.